package/nix/install

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+{ # Prevent execution if this script was only partially downloaded
+  set -e
+
+  GREEN='\033[0;32m'
+  RED='\033[0;31m'
+  NC='\033[0m'
+
+  NIX_POST_INSTALL_MESSAGE="To ensure that the necessary environment variables are set, either log in again, or type
+
+  . $HOME/.nix-profile/etc/profile.d/nix.sh
+
+in your shell."
+
```

Comment:
I don't htink we need to store this script here, instead we store it in K and point it at this one to get `kup`. That's easier build-system wise, and this script shouldn't change so often anyway.